.gitignore

```diff
@@ -7,6 +7,7 @@ node_trace*
 .awcache/
 .clinic
 .eslintcache
+public
```

Comment:
Should we add `public` to gitignore?

Reply:
because of the public folder generated to deploy with now